11_FFTBloom/utils.h

```diff
@@ -0,0 +1,34 @@
+#ifndef FFT_BLOOM_UTILS
+#define FFT_BLOOM_UTILS
+
+
+using namespace nbl;
+using namespace core;
+using namespace system;
+using namespace asset;
+using namespace video;
+
+
+static inline asset::VkExtent3D padDimensions(asset::VkExtent3D dimension)
+{
+	for (auto i = 0u; i < 3u; i++)
+	{
+		auto& coord = (&dimension.width)[i];
+		if (coord <= 1u)
+			continue;
+		coord = core::roundUpToPoT(coord);
+	}
```

Comment:
why are you padding all dimensions to PoT!? 

the last one (last one that is not >1) doesn't need to be padded!

Reply:
also there's no switch for whether its a real signal or not

Reply:
still remains a TODO